src/processing/aggregate_counts.py

```diff
@@ -0,0 +1,29 @@
+
+from pyspark.sql import functions as F
+
+def get_aggregate_counts(df, counting_col, alias_name):
+    '''
+    Function to get counts in dataframe 
+    '''
+    df = (df
+        .agg(
+            F.countDistinct(counting_col).alias(alias_name)
+        )
+    )
```

Comment:
```suggestion
    df = (
        df
        .agg(
            F.countDistinct(counting_col).alias(alias_name)
        )
    )
```

Reply:
Does the above seem overkill? To me it overcomplicated what isn't actually that complicated a line:
`df = df.agg(F.countDistinct(counting_col).alias(alias_name))`

We don't have to break things across lines every time.